src/ui/components/layout/ResponsivePageLayout/ResponsivePageLayout.scss

```diff
@@ -19,7 +19,7 @@
     }
   }
 
-  @media screen and (min-width: 250px) and (max-width: 425px) {
+  @media screen and (min-width: 250px) and (max-width: 389px) {
```

Comment:
Changing this value is going to affect all pages using this layout. This means we should be taking screenshots of the entire app in all devices to ensure everything is showing up as expected.

I believe the main problem is the fact that `ResponsivePageLayout` and `ScrollablePageLayout` have different breakpoints. 

Thinking about this again I would say please revert this change and I will open [another ticket](https://cardanofoundation.atlassian.net/browse/DTIS-884) for you so you can make these adjustments aligning the breakpoints to the same value between the two layouts and make sure all pages in the app will look ok across all devices.

![10052024122738](https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/8188a0cc-5f10-479e-9a59-4672325c816d)